src/utils/communitySbt/helpers.ts

```diff
@@ -80,7 +81,9 @@ export async function geckoEthToUsd(apiKey: string) : Promise<number> {
           `https://pro-api.coingecko.com/api/v3/simple/price?x_cg_pro_api_key=${apiKey}&ids=ethereum&vs_currencies=usd`,
         );
         return data.data.ethereum.usd;
-      } catch (error) {}
+      } catch (error) {
+          Sentry.captureException(error);
```

Comment:
What is the error message in this case? We haven't created a human readable message for some reason in this case, so I am defaulting to the "error" object in hope it will contain enough information for us. If not we can always revisit this piece of code.